server/src/main/proto/spine/server/entity/entity.proto

```diff
@@ -48,19 +48,28 @@ message EntityRecord {
     // The version of the entity.
     base.Version version = 3;
 
-    // The visibility flags of the entity.
-    Visibility visibility = 4;
+    // The lifecycle flags of the entity.
+    LifecycleFlags lifecycle_flags = 4;
 }
 
-// The visibility flags of an entity.
-//
-// If one of the flags is set to `true` the entity becomes “invisible” to regular queries.
-//
-message Visibility {
+// The lifecycle flags of an entity.
+message LifecycleFlags {
```

Comment:
It's a pity we have to duplicate `LifecycleFlags` proto message and `LifecycleFlagField` class structure.

Can we avoid it somehow? ATM, we'll have to append the same changes to these two classes if any new flags are introduced.